### ISSUE TITLE
Required files for Sentinel 2

### ISSUE THREAD

**Issue by evan-blaisdell** [GERSL/Fmask4#4]
What are the bare minimum required files to run Fmask, version 4, on a Sentinel 2 tile? Is the entire SAFE structure necessary?

**Comment by qsly09** [comment]
Yes, it requires the entire SAFE structure having Level-1C (TOA) products.
